src/Search/SearchOptions.php

```diff
@@ -0,0 +1,177 @@
+<?php
```

Comment:
I really like the idea of having a Util class instead of a function in the Controller like what i did last time 👍